src/Core/Services/LiteDbStorageService.cs

```diff
@@ -68,7 +68,7 @@ public async Task SaveAsync<T>(string key, T obj)
         public async Task RemoveAsync(string key)
         {
             await InitAsync();
-            _collection.Delete(i => i.Id == key);
+            _collection.DeleteMany(i => i.Id == key);
```

Comment:
Did Delete go away in v5? This is an Id delete, so should only ever be one record deleted with it.

Reply:
Delete no longer accepts the lambda expression but DeleteMany does.  I'll need to convert the expression into a `LiteDB.BsonValue` to use Delete.

Reply:
Rethinking this.. If I'm reading the expression correctly it can technically return more than one value (even if there is only one match).  Seems like they changed the method name to be more explicit, but the old `Delete` was performing the same action that `DeleteMany` is now.  How do you want to narrow down the correct ID to pass to the new `Delete`?

Reply:
`DeleteMany` is fine.